### ISSUE TITLE
Queries with Limit resulting in sql error

### ISSUE THREAD

**Issue by peekolo** [antradar/sqldash#1]
SQLDash appends limit to the end of the query. If the executed query contains a limit, it results in an error
![Screenshot 2023-03-14 at 12 18 13 PM](https://user-images.githubusercontent.com/43511539/225069962-fdcd12cd-891f-40e5-b1ff-c58dc18ab92c.png)

**Comment by peekolo** [comment]
I've put a temporary patch in as
```
$limitfound=0;
if(preg_match_all('/limit\s*(\d+),\s*\d+$/',trim($query)) || preg_match_all('/limit\s*(\d+)$/',trim($query))){
    $limitfound=1;
}
```
and check for `$limitfound` later on before appending
```
if (!$limitfound && in_array($SQL_ENGINE,array('MySQL','MySQLi','ClickHouse'))) $query.=" limit $start,$perpage ";
```

**Comment by schien-dong** [comment]
Thank you peekolo for pointing out this issue. Your patch is sound. I will publish the changes soon. There is no need for a PR. But I will mention your contribution if you like? Let me know. Thanks.

**Comment by schien-dong** [comment]
Fixed in 6d71946..4856f92
The "limit X,Y" and "limit X" should not be simply bypassed because that would break the pagination. Instead:
- limit X is rewritten to limit 0,X
- limit X, Y will set the per_page to Y and reverse-calculate the initial page number IF the page is not set in $_GET

Try limit 1, limit 15,5 to see the results.
